js/index.js

```diff
@@ -63,6 +63,42 @@ function displayProjects() {
   projectCards.appendChild(cards);
 }
 
+const handleSubmit = (event) => {
+  event.preventDefault();
+  const nameError = document.getElementById('name-error');
+  const emailError = document.getElementById('email-error');
+  const emailIcnorect = document.getElementById('emile-incorect');
+  const msgError = document.getElementById('msg-error');
+  const name = document.getElementById('name-input').value;
+  const email = document.getElementById('email-input').value;
+  const msg = document.getElementById('msg-input').value;
+  let error = false;
+  if (name.length === 0) {
+    nameError.innerHTML = 'Name is required';
+    error = true;
+  }
+  if (email.length === 0) {
+    emailError.innerHTML = 'Email is required';
+    error = true;
+  }
+  if (!email.match('[a-z0-9]+@[a-z]+.[a-z]{2,3}')) {
+    emailIcnorect.innerHTML = 'Email is incorect';
+    error = true;
+  }
+  if (email.match('[A-Z0-9]+@[A-Z]+.[A-Z]{2,3}')) {
+    emailIcnorect.innerHTML = 'Email is incorect';
+    error = true;
+  }
+  if (msg.length === 0) {
+    msgError.innerHTML = 'Message is required';
+    error = true;
+  }
+  if (!error) {
+    const form = document.getElementById('contact-form');
+    form.submit();
+  }
```

Comment:
- [  REQUIRED ] Hi, the form still gets submitted even when there is a capital letter in the email. You are required to make sure that the email must be lowercased before the form gets submitted. Kindly make sure that you check only lowercased email gets submitted. 

### See the test email I sent 

![image](https://user-images.githubusercontent.com/50941074/174459829-cc561de3-934a-420b-a9c0-9b11657fb4ab.png)